frontend/src/components/smart/WeaponGrid.vue

```diff
@@ -272,6 +263,15 @@ export default Vue.extend({
       this.$emit('weapon-filters-changed');
     },
 
+    priceSortText(type: string): string {
```

Comment:
same as above, use the sorts array instead of this so we don't need this function - I'm not sure why it's here twice.